components/common/ConnectYouTubeButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { Youtube } from "lucide-react";
+import { youtubeLogin } from "@/lib/youtube/youtubeLogin";
+import { Button } from "@/components/ui/button";
+
+interface ConnectYouTubeButtonProps {
+  accountId?: string;
+  className?: string;
+  size?: "sm" | "md" | "lg";
+  disabled?: boolean;
+}
+
+const sizeMap: Record<"sm" | "md" | "lg", "sm" | "lg" | "default"> = {
+  sm: "sm",
+  md: "default",
+  lg: "lg",
+};
```

Comment:
Could you delete this size map and directly pass the `size` param to the Button component?